pkg/metrics/metrics.go

```diff
@@ -22,30 +22,43 @@ import (
 )
 
 const (
-	metricGardenProjectsSum = "garden_projects_total"
-	metricGardenUsersSum    = "garden_users_total"
+	metricGardenProjectsSum    = "garden_projects_total"
+	metricGardenSeedsSum       = "garden_seeds_total"
+	metricGardenProjectsStatus = "garden_projects_status"
+	metricGardenUsersSum       = "garden_users_total"
+
+	// Seed metric
+	metricGardenSeedInfo      = "garden_seed_info"
+	metricGardenSeedCondition = "garden_seed_condition"
 
 	// Shoot metric (available also for Shoots which act as Seed)
-	metricGardenShootInfo             = "garden_shoot_info"
-	metricGardenShootCondition        = "garden_shoot_condition"
-	metricGardenShootOperationState   = "garden_shoot_operation_states"
-	metricGardenShootNodeMaxTotal     = "garden_shoot_node_max_total"
-	metricGardenShootNodeMinTotal     = "garden_shoot_node_min_total"
-	metricGardenShootResponseDuration = "garden_shoot_response_duration_milliseconds"
+	metricGardenShootInfo              = "garden_shoot_info"
+	metricGardenShootCondition         = "garden_shoot_condition"
+	metricGardenShootOperationState    = "garden_shoot_operation_states"
+	metricGardenShootOperationProgress = "garden_shoot_operation_progress"
```

Comment:
`garden_shoot_operation_progress` should be renamed to express that the value is a percent. For example, `garden_shoot_operation_percent_progress`. It might make sense to combine this metric with `garden_shoot_operation_states`. Any thoughts @dkistner?

Reply:
I think the metrics express different things. 

`garden_shoot_operation_states` represents the current status (`Running|Succeded|Error ...`) of an operation (`Create|Reconcile|Delete`).

`garden_shoot_operation_progress ` represents the current progress of an operation (`Create|Reconcile|Delete`) e. g. 100. I agree we should rename the metric to be conform with the metric naming conventions e. g. `garden_shoot_operation_progress_percentage`.

I think we should have both metrics, because those metric is a nice addition to the existing one.

Reply:
done